src/extensions/actions/ActionsRuleCreationModal.tsx

```diff
@@ -370,13 +393,23 @@ export const NeoCustomReportActionsModal = ({
                               popupIcon={<></>}
                               style={{ minWidth: 250 }}
                               onInputChange={(event, value) => {
-                                updateRuleField(index, 'value', value);
+                                if (type == 'bar') {
```

Comment:
also `onChange` and `onInputChange` in seperate functions, with explanation why we set `field` too.

Reply:
Done